mask_topic: "/yolov7/Detection"
depth_topic: "/camera/aligned_depth_to_color/image_raw"
color_topic: "/camera/color/image_raw"
camera_info_topic: "camera/aligned_depth_to_color/camera_info"
publish_topic: "/segpc_yolov7/points"
publish_rate: 10